inspire/base/static/js/dropdown-menu/dropdown-menu.js

```diff
@@ -0,0 +1,150 @@
+        arrowDefaultState("#fourth-arrow");
+        arrowDefaultState("#fifth-arrow");
+        arrowDefaultState("#sixth-arrow");
+        arrowDefaultState("#seventh-arrow");     
+      
+
+        function activateSubmenu(row) {
+            var $row = $(row),
+                submenuId = $row.data("submenuId"),
+                $submenu = $("#" + submenuId),
+                height = $menu.outerHeight(),
+                width = $menu.outerWidth();
+            // Show the submenu
+            $submenu.css({
+                display: "block",
+                top: -1,
+                left: width - 3  // main should overlay submenu
+                //height: height - 4  // padding for main dropdown's arrow
+              });
+            }
+
+        function deactivateSubmenu(row) {
+            var $row = $(row),
+                submenuId = $row.data("submenuId"),
+                $submenu = $("#" + submenuId);
+            // Hide the submenu and remove the row's highlighted look
+            $submenu.css("display", "none");
+            //$row.find("a").removeClass("maintainHover");
+          }
+
+        $(".dropdown-menu li").click(function(e) {
+            e.stopPropagation();
+        });
+
+        $(document).click(function() {
+          // Simply hide the submenu on any click. Again, this is just a hacked
+          // together menu/submenu structure to show the use of jQuery-menu-aim.
+          $(".popover").css("display", "none");
+         });
+
+      //Function for opening both parts on click
+        /*$(".dropdown-toggle").click(function() {
+           $("#jobs").css({"display":"block","top": "-1px", "left": "733px","position":"absolute"});
+          });*/
+        }
+        else {
+          $(document).click(function() {
+          $(".move-down").css({"margin-top":"0px"});
+         });
+
+          $(".popover").css({"display":"none"});
+          $(".dropdown").removeClass('open');
+          $(".dropdown-toggle").unbind("click");         
+          $(".first-submenu-item, .second-submenu-item, .third-submenu-item," +
+            ".fourth-submenu-item, .fifth-submenu-item, .sixth-submenu-item, .seventh-submenu-item").unbind();
+         
+          $(".move-down").animate({"margin-top":"0px"}, "fast");
+    
+          responsiveDropdown("#first-element","#jobs","#first-arrow");
+          responsiveDropdown("#second-element","#tools","#second-arrow");
+          responsiveDropdown("#third-element","#conferences","#third-arrow");
+          responsiveDropdown("#fourth-element","#journals","#fourth-arrow");
+          responsiveDropdown("#fifth-element","#experiments","#fifth-arrow");
+          responsiveDropdown("#sixth-element","#institutions","#sixth-arrow");
+          responsiveDropdown("#seventh-element","#links-and-stats","#seventh-arrow");     
+        }
+      }
+    });
+  
+
+function arrowDefaultState(id) {
+  $(id).attr('src','img/right-arrow.png');
+  $(id).unbind("click")
```

Comment:
The preferred method is to use [on()](http://api.jquery.com/on/) and [off(http://api.jquery.com/off/)] methods